packages/storybook-vue/stories/components/sidebar-navigation/SidebarNav.vue

```diff
@@ -2,11 +2,12 @@
 export default {
   name: 'scale-sidebar-nav',
   props: {
-    ariaLabel: { type: String },
+    ariaLabelSiderbarNav: { type: String },
```

Comment:
prop description need to be added for all of these props: 
https://github.com/telekom/scale/blob/8a04a5e8c0bb3b72a2fbb90822600f6b8feefc06/packages/components/src/components/sidebar-nav/readme.md?plain=1#L12